plotter/views.py

```diff
@@ -104,14 +104,14 @@ def delete_profile(request):
     return response
 
 
-# @view_config(route_name="secret")
-# @check_export
-# def secret(request):
-    # fn = db.secret_file("%(name)s%(suffix)s" % request.matchdict)
-    # return FileResponse(fn, request=request)
+@view_config(route_name="secret")
+@check_export
+def secret(request):
+    fn = db.secret_file("%(name)s%(suffix)s" % request.matchdict)
+    return FileResponse(fn, request=request)
```

Comment:
I had to uncomment this in order to get the server to send scatterplots to the old chrom viewer pages